docs/sources/guides/whats-new-in-v6-5.md

```diff
@@ -35,6 +38,65 @@ Grafana 6.5 comes with a lot of new features and enhancements.
+
+In Grafana 6.5 or higher, you’re able to monitor a dynamic list of metrics by using the asterisk (\*) wildcard for any one or multiple dimension values.
+
+In the example below, all metrics in the namespace `AWS/EC2` with a metric name of `CPUUtilization` and ANY value for the `InstanceId` dimension are queried. This can help you monitor metrics for AWS resources, like EC2 instances or containers. For example, when new instances get created as part of an auto scaling event, they will automatically appear in the graph without you having to track the new instance IDs. This capability is currently limited to retrieving up to 100 metrics. You can click on `Show Query Preview` to see the search expression that is automatically built to support wildcards. To learn more about search expressions, visit the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html).
+
+By default, the search expression is defined in such a way that the queried metrics must match the defined dimension names exactly. This means that in the example below only metrics with exactly one dimension with name ‘InstanceId’ will be returned.
+
+You can untoggle `Match Exact` to include metrics that have other dimensions defined. Disabling ‘Match Exact’ also creates a search expression even if you don’t use wildcards. We simply search for any metric that match at least the namespace, metric name, and all defined dimensions.
+
+!!image
+
+#### Deep linking from Grafana panels to the CloudWatch console
+
+Left clicking a time series in the panel shows a context menu with a link to `View in CloudWatch console`. Clicking that link will open a new tab that will take you to the CloudWatch console and display all the metrics for that query. If you are not currently logged in to the CloudWatch console, the link will forward you to the login page. The provided link is valid for any account but will only display the right metrics if you are logged in to the account that corresponds to the selected data source in Grafana.
+
+!!image
+
+This feature is not available for metrics that are based on math expressions.
+
+#### Improved feedback when throttling occurs
+
+If the [limit of the GetMetricData API](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html) is reached (either the transactions per second limit or the data points per second limit), a throttling error will be returned by the CloudWatch API. Throttling limits are defined per account and region, so the alert modal will indicate which data source got throttled in which region. A link to request a limit increase will be provided for the affected region, but you will have to log in to the correct account. For example, for us-east-1, a limit increase can be requested [here](https://console.aws.amazon.com/servicequotas/home?region=us-east-1#!/services/monitoring/quotas/L-5E141212).
+
+#### Angular to React migration
```

Comment:
Not sure if this is needed as it is a technical change that most users won't/don't need to care about.

Reply:
I kind of agree, but apparently this has been a pain for a lot of users so I think it might be worth mentioning this in the what's new post.

Reply:
Agree mentioning react and angular is probably too technical. Would recommend focusing on what improvements have been made regarding user experience/ux, look and feel etc. The sentence "basically the same as before, but it has a new look and feel to it" doesn't really give a hint of why this have been a pain before. Maybe change the section name to Improved UX or something similar?